windows/windows-impl/src/main/java/com/duckduckgo/windows/impl/waitlist/ui/WindowsWaitlistActivity.kt

```diff
@@ -133,25 +136,42 @@ class WindowsWaitlistActivity : DuckDuckGoActivity() {
 
     private fun renderJoinedQueue() {
         binding.waitListButton.gone()
-        binding.footerDescription.gone()
-        binding.lookingForMacVersionButton.show()
         binding.codeFrame.gone()
         binding.shareImage.gone()
+        binding.waitlistNotifyMeContainer.show()
+        binding.lookingForMacVersionButton.show()
+        binding.footerDescription.gone()
         binding.headerImage.setImageResource(com.duckduckgo.mobile.android.R.drawable.ic_list)
         binding.statusTitle.text = getString(R.string.windows_waitlist_on_the_list_title)
-        binding.waitlistDescription.text = getText(R.string.windows_waitlist_on_the_list_notification)
+        binding.waitlistDescription.text = getJoinedQueueDescriptionText(binding.waitlistNotifyMe.isVisible)
+        binding.waitlistNotifyMe.setOnVisibilityChange(
```

Comment:
@malmstein as I mentioned, we change the description copy based on the notifyme component visiblity.